site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1688,9 +1688,15 @@ components:
         name: filterJson
         in: query
         description: |
+           
+          Using the following filter filterJson={"field": "/gateways","value": false}, all devices can be listed
+          excluding gateways. With the filter filterJson={"field": "/gateways","value": true} it is possible to
+          list only gateways.
+         
```

Comment:
can you please move this to the `examples` below?
Does this also work with the MongoDB based registry?

Reply:
Done. 
No gateway filters don't work with MongoDB.

Reply:
why not?